stripper/ze_paranoid_rezurrection_csgo1.cfg

```diff
@@ -27,4 +28,57 @@ modify:
 	{
 		"OnMapSpawn" "consolaCommandmp_buytime 22-1"
 	}
-}
\ No newline at end of file
+}
+
+// Changes by "Berke".
+// Restore "Destroyer".
```

Comment:
I'm usually not a stickler for enforcing formatting, but excluding directives and using the fall through like this can get quite confusing and isn't done elsewhere in the repo.

Reply:
> Also can you show the reference destroyer implementation from CS:S paranoid? Would like to verify this is a valid port.

Here is the video from CS:S, console button only spawns the Destroyer.
https://streamable.com/s8lnvj

It is spawned by pressing +use on the console in the LMS room on the original map, it can be hold by Gravity Gun to kill zombies that touch it. It is removed in the CS:GO port, porter changed the button to spawn the Holy Grenade instead, but Destroyer particle was still packed, with my stripper the button now spawns both the Holy Grenade and the Destroyer.

> I'm usually not a stickler for enforcing formatting, but excluding directives and using the fall through like this can get quite confusing and isn't done elsewhere in the repo.

I am guessing you mean the lack of "modify:"? Anyway I added them now.